troposphere/static/js/components/projects/detail/resources/InstanceActionButtons.jsx

```diff
@@ -59,6 +63,14 @@ export default React.createClass({
                 );
                 linksArray.push(
                     <Button style={style}
+                        key="Resize"
+                        icon="repeat"
```

Comment:
@xuhang57 - for context (and connecting conversations), what Tharon is referring to is a move to using the `api/v2/instances/:alias/actions` endpoint to determine what are _valid_ actions. This is in a pending pull request targeted to the Xylotomous-Xenops release (it was slated for Whimsical-Wyvern, but has slipped).